backend/src/main/java/de/sambam/gamenighttracker/controller/UserController.java

```diff
@@ -24,11 +22,19 @@ public UserController(UserService userService) {
         this.userService = userService;
     }
 
-    @GetMapping({"/games"})
-    public List<Game> gameList() {
-        return userService.listAllGames("1")
-                .orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "User not found"));
+    @GetMapping("/games")
+    public List<Game> getGameList() {
+        return userService.listAllGames("1");
```

Comment:
why are you calling your method with a constant parameter `"1"`? 
if you use this value for all methods you can also include it as a `private static final String id = "1";`

Reply:
it's for this state of the project, basically my one and only user for now